app/src/main/java/com/release/keyneez/data/model/response/ResponseGetSearchDto.kt

```diff
@@ -0,0 +1,18 @@
+package com.release.keyneez.data.model.response
+
+import kotlinx.serialization.SerialName
+
+@kotlinx.serialization.Serializable
+data class ResponseGetSearchDto(
```

Comment:
조금 더 구체적이어도 될 것 같은데 어떤가요?
```suggestion
data class ResponseGetSearchResultDto(
```

Reply:
너무 네이밍이 길어지는 것 같아서 머뭇거렸는데 말씀하신대로 수정하도록 하겠습니다!